<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewHomeWork</class>
 <widget class="QWidget" name="NewHomeWork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>447</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lPaperName">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Paper Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="lSelectSubject">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Subject</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEPaperName">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lDuration">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Duration</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEDuration">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="gBSelectType">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>371</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Select Type</string>
   </property>
   <widget class="QRadioButton" name="rBMcq">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mcq</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBEssay">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Essay</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBEssayMcq">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mcq and Essay</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pBCreatePaper">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Paper</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBClose">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>380</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="tWSelectSubject">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>371</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pBClose</sender>
   <signal>clicked()</signal>
   <receiver>NewHomeWork</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>715</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>901</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
